packages/connector-chromium/package.json

```diff
@@ -0,0 +1,88 @@
+{
+  "ava": {
+    "failFast": false,
+    "files": [
+      "dist/tests/**/*.js"
+    ],
+    "timeout": "1m"
+  },
+  "dependencies": {
+    "@hint/utils": "^1.0.0",
+    "puppeteer-core": "^1.14.0"
+  },
+  "description": "hint connector for Chromium based browsers",
+  "devDependencies": {
+    "@hint/utils-create-server": "^3.0.0",
+    "@types/is-ci": "^1.1.0",
+    "@types/lodash": "^4.14.123",
+    "@types/node": "^11.13.4",
+    "@types/puppeteer-core": "^1.9.0",
+    "@types/sinon": "^7.0.11",
+    "@typescript-eslint/eslint-plugin": "^1.6.0",
+    "@typescript-eslint/parser": "^1.6.0",
+    "ava": "^1.4.1",
+    "cpx": "^1.5.0",
+    "eslint": "^5.15.1",
+    "eslint-plugin-import": "^2.16.0",
+    "eslint-plugin-markdown": "^1.0.0",
+    "hint": "^4.5.0",
+    "lodash": "^4.17.11",
+    "npm-run-all": "^4.1.5",
+    "nyc": "^14.0.0",
+    "rimraf": "^2.6.3",
+    "sinon": "^7.3.2",
+    "typescript": "^3.4.3"
+  },
+  "engines": {
+    "node": ">=8.0.0"
+  },
+  "files": [
+    "dist/src"
+  ],
+  "homepage": "https://webhint.io/",
+  "keywords": [
+    "connector",
+    "chromium",
+    "edge",
+    "microsoft edge",
+    "chrome",
+    "google chrome",
+    "hint",
+    "hint-connector-chromium",
+    "webhint"
+  ],
+  "license": "Apache-2.0",
+  "main": "./dist/src/connector.js",
+  "name": "@hint/connector-chromium",
+  "nyc": {
+    "branches": 30,
+    "extends": "../../.nycrc",
+    "functions": 70,
+    "include": [
+      "dist/src/**/*.js"
+    ],
+    "lines": 70,
+    "statements": 70
```

Comment:
That's a quite a few overrides of our default coverage settings. Any particular reason why?

Reply:
copy paste from `connector-chrome`